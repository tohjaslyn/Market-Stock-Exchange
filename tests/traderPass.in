add trader1 100
add trader2 10
traders
exit
