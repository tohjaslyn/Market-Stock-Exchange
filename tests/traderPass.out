$ Success.
$ Success.
$ trader1
trader2
$ Have a nice day.
